Backend/src/app/models/user.ts

```diff
@@ -31,7 +31,7 @@ const userSchema = new Schema({
     },
     role: {
         type: String,
-        default: 'user' //defaults to normal user
+        default: 'user'  //defaults to normal user
```

Comment:
My bad, missed one comment from last review

```suggestion
        default: UserRole.User  //defaults to normal user
```